include/cpu/tb.h

```diff
@@ -235,4 +237,23 @@ static inline void tb_add_jump(TranslationBlock *tb, int n, TranslationBlock *tb
     }
 }
 
-#endif
+#ifdef CONFIG_SYMBEX
+/* Get the address of the instruction after the one starting at the given address.
+   Useful for variable-length instruction sets. */
+static inline uint64_t tb_get_next_instruction_addr(TranslationBlock *tb, uint64_t addr) {
```

Comment:
I think semantically it would make more sense to call this tb_get_instruction_size(), because there may be no instruction at the address returned by tb_get_next_instruction_addr(), e.g., in case of an unconditional jump.